examples/quickstart/jupyter-notebooks/druid-visualization.ipynb

```diff
@@ -0,0 +1,620 @@
+   "source": [
+    "First, call `output_notebook()` to set Bokeh to output plots inline within the notebook. This lets you view your plot inline. You can also use Bokeh's `output_file` function to write your plot to an HTML file. If you want to experiment with `output_file` add it to the list of imports."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d824a82f-9e71-43c4-b289-78d4970c5cfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_notebook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5859d743-a5a7-445c-a06f-89b8c30856ed",
+   "metadata": {},
+   "source": [
+    "There are several ways to use Bokeh with a DataFrame. In this case, make a list of channels to serve as the x axis of our plot. For the y axis,  divide the total additions by 100000 for ease of display."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70b40bfc-106d-462c-a5a1-51d0da4c4498",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "channels = additions_df.channel.to_list()\n",
+    "total_additions = [x / 100000 for x in additions_df.additions.to_list()]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efc5f142-8b68-4430-a666-2ed961d732b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "channels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f619fd3-6a01-4dc6-b8eb-81e4b4ae1a31",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "total_additions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b430b4c-df81-419d-a81e-63bb0150031b",
+   "metadata": {},
+   "source": [
+    "Next, initialize your Bokeh plot (figure) with some basic configurations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07644ccd-1f0c-4a50-8929-82559f9bf803",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " # Create a new plot with a title. Set the size in pixels\n",
+    "bar_plot = figure(height=500, width=750, x_range=channels,  title=\"100000 x Additions\",\n",
+    "           toolbar_location=None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0a01f4c-6f35-4eb7-9263-4f6ed6c68223",
+   "metadata": {},
+   "source": [
+    "Now, configure the rederer for the vertical bars on the plot:\n",
+    "- Set the x axis to `channels`, the list of channels.\n",
+    "- Set the `top` coordinate that determines the bar height to `totals`.\n",
+    "- For a splash of color, set the `color` to `Spectral10`. \n",
+    "\n",
+    "Note that palettes in Bokeh are lists of colors. Bokeh expects the list length to equal the list length of the data dimensions -- in this case 10 colors.\n",
+    "\n",
+    "See the Bokeh docs for more information on [vertical bars](https://docs.bokeh.org/en/latest/docs/reference/plotting/figure.html#bokeh.plotting.figure.vbar) and [palettes](https://docs.bokeh.org/en/latest/docs/reference/palettes.html)."
```

Comment:
Just checking, there isn't supposed to be an output for this cell right? I thought mine was not working properly but then figured it might just be a meaningless output:

![image](https://user-images.githubusercontent.com/7747997/211445619-f5d0ebd6-0757-4453-93e3-edb29f7a6201.png)

Reply:
This is normal behavior